stores/planship.ts

```diff
@@ -0,0 +1,258 @@
+import { defineStore, skipHydrate } from 'pinia'
+
+import { useUserStore } from '@/stores/user'
+import { useProjectsStore } from '@/stores/projects'
+
+import { Planship } from '@planship/fetch'
+
+async function getAccessToken() {
+  return fetch('/api/planshipToken').then((response) => response.text())
+}
+
+function getApiUrl() {
+  return typeof window === 'undefined' ? useRuntimeConfig().public.serverPlanshipApiUrl : useRuntimeConfig().public.clientPlanshipApiUrl
```

Comment:
You can/should use `process.server`/`process.client` for this I believe. We should consider if there's a better way to handle SSR in general.